.eslintrc

```diff
@@ -0,0 +1,137 @@
+{
+  "parser": "babel-eslint",
+
+  "ecmaFeatures": {
+    "modules": true,
+    "experimentalObjectRestSpread": true
+  },
+
+  "env": {
+    "browser": false,
+    "es6": true,
+  },
+
+  "rules": {
+    // specify the maximum depth that blocks can be nested
+    "max-depth": [0, 4],
+    // specify the maximum length of a line in your program
+    "max-len": [0, 80, 4],
```

Comment:
ok, got it:
 zero is about rule priority (should it be 2 here -  turn the rule on as an error)
80 is limitation
4 - each tab is 4 characters